Assets/Mirror/Authenticators/TimeoutAuthenticator.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using UnityEngine;
+
+namespace Mirror.Authenticators
+{
+    /// <summary>
+    /// An authenticator that disconnects connections if they don't
+    /// authenticate within a specified time limit.
+    /// </summary>
+    [AddComponentMenu("Network/Authenticators/TimeoutAuthenticator")]
+    public class TimeoutAuthenticator : NetworkAuthenticator
+    {
+        public NetworkAuthenticator authenticator;
+        
+        public int timeout = 60;
```

Comment:
why is this int?

Reply:
It was a byte in mine, default o which disabled the timeout.

Reply:
I can make it a short,   but a byte seems to limited.   a byte would be 256 seconds = 4 minutes maximum.

I could also make it a float.   Any particular preference?